lib/mixpanel-node.js

```diff
@@ -54,7 +59,11 @@ var create_client = function(token, config) {
         }
 
         var request_options = {
-            host: 'api.mixpanel.com',
+
+            // Split host, to host and port, for http.get
+            host: metrics.config.host.split(':')[0],
+            port: metrics.config.host.split(':')[1] ? metrics.config.host.split(':')[1] : 80,
```

Comment:
Maybe it would nice to do this when setting config in `metrics.set_config()`, so your server doesn't have to re-execute this extra string manipulation on every mixpanel request?